packages/koenig-lexical/src/plugins/DragDropPastePlugin.jsx

```diff
@@ -0,0 +1,111 @@
+
+// TODO - replace the next 2 isMimeType and mediaFileReader with  @lexical/utils library when released in next `not nightly` version of Lexical - currently not available in latest published Lexical version.
+// NB added minor adjustment to mediaFileReader to adapt to existing uploaders, 
+// both on Ghost and Demo - we should adapt those functions to meet Lexicals output which is an array like [{file: fileData, result: base64 String}] )
+// source https://github.com/facebook/lexical/blob/main/packages/lexical-utils/src/index.ts
+
+function isMimeType(file, acceptableMimeTypes) {
+    const mimeType = file.type;
+    let key = Object.keys(acceptableMimeTypes).find(k => acceptableMimeTypes[k].includes(mimeType));
+    return key;
+}
+  
+function mediaFileReader(files, acceptableMimeTypes) {
+    const filesIterator = files[Symbol.iterator]();
+    return new Promise((resolve, reject) => {
+        const processed = [];
+        let node;
+        const handleNextFile = () => {
+            const {done, value: file} = filesIterator.next();
+            if (done) {
+                return resolve({processed, node});
+            }
+            const fileReader = new FileReader();
+            fileReader.addEventListener('error', reject);
+            fileReader.addEventListener('load', () => {
+                const result = fileReader.result;
+                if (typeof result === 'string') {
+                    processed.push(file);
+                }
+                handleNextFile();
+            });
+            const nodeType = isMimeType(file, acceptableMimeTypes);
+            if (nodeType) {
+                fileReader.readAsDataURL(file);
+                node = nodeType;
+            } else {
+                console.error('unsupported file type'); // eslint-disable-line no-console
+                handleNextFile();
+            }
+        };
+        handleNextFile();
+    });
+}
+
+async function getListofAcceptableMimeTypes(editor) {
+    const nodes = getEditorCardNodes(editor);
+    let acceptableMimeTypes = {};
+    for (const [nodeType, node] of nodes) {
+        if (nodeType && node.mimeTypes) {
+            acceptableMimeTypes[nodeType] = node.mimeTypes;
+        }
+    }
+    return {
+        acceptableMimeTypes
+    };
+}
+
+function DragDropPastePlugin() {
+    const [editor] = useLexicalComposerContext();
+
+    const handleDrag = React.useCallback((e) => {
+        e.preventDefault();
+        e.stopPropagation();
+    }, []);
+
+    const handleDrop = React.useCallback(async (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        if (e.dataTransfer.files) {
+            editor.dispatchCommand(DRAG_DROP_PASTE, e.dataTransfer.files);
+        }
+    }, [editor]);
+
+    React.useEffect(() => {
+        // TODO: consuming app should be able to configure the element where they want to listen to the drag and drop events
+        const dropTarget = document.querySelector('.gh-koenig-editor') || document;
```

Comment:
I think for now the safest route will be to use `editor.getRootElement()` as the default in all cases so that multiple koenig editors can exist on-screen.